frappe/commands/utils.py

```diff
@@ -894,6 +896,9 @@ def run_ui_tests(
 	if ci_build_id:
 		formatted_command += f" --ci-build-id {ci_build_id}"
 
+	if spec:
+		formatted_command += f" --spec {spec}"
```

Comment:
You don't need this.. You can pass arbitrary arguments to cypress. 
https://github.com/frappe/frappe/commit/4944246a0db94ee598d16d5689c2fefb8995df3e

Reply:
@ankush Cool! Thanks for the pointer!